### ISSUE TITLE
Report an error

### ISSUE THREAD

**Issue by dtt355** [XLIU430/compsys723#1]
Sorry to interrupt, I saw your problem under AI4HealthUOL/ SSD-ECG project, I also encountered RuntimeError: mat1 and mat2 shapes cannot be multiplied (6x44 and 71x128). Have you corrected it? Can you give me some advice?Thank you!

**Comment by XLIU430** [comment]
Yes, I finally made the code running, and it is be able to train now. I modified a lot of code. For this problem, I modified the configure file. Can you send your email. So I can send all the modified python script as a zip file to you.

**Comment by dtt355** [comment]
Thank you very much for your willingness to give me all the changed python scripts. Thank you very, very much！My email：du_tian2021@163.com
